rfc-192-use-codeowners-in-all-repos.md

```diff
@@ -0,0 +1,77 @@
+# Use `CODEOWNERS` in all repositories
+
+## Summary
+
+We should configure our repositories with
+[GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+or [GitLab](https://docs.gitlab.com/ee/user/project/code_owners.html)
+`CODEOWNERS` files, to ensure that issues and PRs receive a timely response, and
+that our code is well maintained.
+
+## Problem
+
+`dxw` has a large number of repositories to maintain. At the time of writing,
+GitHub has 260 repositories that are not archived and not forked from elsewhere,
+151 of which are public. It is easy for new PRs and issues to be lost in the
+noise of notifications. For maintainers, it can be difficult to keep up with
+changes in these RFCs, especially for internal code which takes a lower priority
+than client products. For developers, it can be difficult to identify someone to
+review code changes and respond to issues.
+
+## Proposal
+
+For the purposes of this RFC, the term _active repository_ means a repository
+that is:
+
+1. within the `dxw` organisation on GitHub or the equivalent on GitLab,
+1. not a fork of a repository from outside the organisation; and
+1. not archived.
```

Comment:
I suggest this list also has `1. not wholly relating to a client`. Client projects MAY use codeowners, but they're also self-managing. This might complicate things with GovPress, though.

Reply:
OK, I think you mean something like: `1. where the maintenance of the repository is entirely within the control of dxw, and not (for example) a client.`?

Reply:
No. I mean things that aren't client projects, because client projects are their own things that have different needs. That's a D+ need anyway.

Reply:
OK, that's a little more complex than I'd realised. Maybe it would be a good idea for repos who "should" conform to all of the maintenance RFCs to have a topic set, like `dxw-maintenance`, which would mean that any automated tools could skip things like D+ client projects?

Reply:
Probably easier to tag the non-dxw stuff, but yes.